tests/tests_fabric/plugins/environments/test_xla.py

```diff
@@ -36,15 +36,18 @@ def test_default_attributes(monkeypatch):
             monkeypatch.setattr(runtime, "world_size", lambda: 1)
             monkeypatch.setattr(runtime, "global_ordinal", lambda: 0)
             monkeypatch.setattr(runtime, "local_ordinal", lambda: 0)
+            monkeypatch.setattr(runtime, "host_index", lambda: 3)
         else:
             from torch_xla.experimental import pjrt
 
             monkeypatch.setattr(pjrt, "world_size", lambda: 1)
             monkeypatch.setattr(pjrt, "global_ordinal", lambda: 0)
             monkeypatch.setattr(pjrt, "local_ordinal", lambda: 0)
+            os.environ["XRT_HOST_ORDINAL"] = "3"
     else:
         from torch_xla import _XLAC
 
+        os.environ["XRT_HOST_ORDINAL"] = "3"
```

Comment:
This could have global effects on other tests. I think the fixture should trigger a "leaking environment variable" pytest error.

Reply:
nit: world size = 1 and node rank = 3 would never be a configuration we see :)